.landscaper/component-descriptor.yaml

```diff
@@ -0,0 +1,12 @@
+component:
+  componentReferences: []
+  name: github.com/gardener/gardener-extension-provider-aws
+  provider: internal
+  repositoryContexts:
+  - baseUrl: eu.gcr.io/sap-gcp-cp-k8s-stable-hub/examples/landscaper/temp
+    type: ociRegistry
```

Comment:
What's this?

Reply:
We have used this component descriptor for local testing. We have removed it.